examples/cookbook/jotai/types.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import { PrimitiveAtom } from 'jotai/vanilla/atom';
+import { useHydrateAtoms } from 'jotai/utils';
+import { RenderOptions as RntlRenderOptions } from '@testing-library/react-native';
+
+type WithInitialValue<Value> = {
+  init: Value;
+};
+type UseHydrateAtomsParams = Parameters<typeof useHydrateAtoms>;
+export type InitialValues = UseHydrateAtomsParams[0];
+export type InitialValue<Value> = [PrimitiveAtom<Value> & WithInitialValue<Value>, Value];
+
+export interface IRenderWithAtomsOptions<T> extends RntlRenderOptions {
```

Comment:
[nit] Let's drop that `I` prefix for TS interfaces.

```suggestion
export interface RenderWithAtomsOptions<T> extends RenderOptions {
```